api/include/opentelemetry/version.h

```diff
@@ -8,6 +8,10 @@
 
 #define OPENTELEMETRY_ABI_VERSION_NO 1
 #define OPENTELEMETRY_VERSION "1.8.2"
```

Comment:
Use # operator for preprocessor to define OPENTELEMETRY_VERSION with the new VERSION macros?

Reply:
Technically possible, but the resulting code, with `OPENTELEMETRY_CONCAT()` and `OPENTELEMETRY_STRINGIFY()`, will be extremely obfuscated.

There is no risk that the string version goes out of sync with major/minor/patch versions, there is a unit test to catch this.